specification/cognitiveservices/Language.AnalyzeText/models/pii.entity.recognition.tsp

```diff
@@ -32,12 +32,63 @@ model PiiTaskParameters is PreBuiltTaskParameters {
   stringIndexType?: StringIndexType = StringIndexType.TextElements_v8;
 
   /** Enumeration of PII categories to be excluded in the response. */
-  @added(Versions.v2023_11_15_preview)
+  @added(Versions.v2024_11_15_preview)
   excludePiiCategories?: Array<PiiCategoriesExclude>;
 
   /** Character to be used for redaction. Default in '*'. */
-  @added(Versions.v2023_11_15_preview)
+  @added(Versions.v2024_11_15_preview)
```

Comment:
We need to remove this from the nov preview

Reply:
removing